print-apps/oereb/config.yaml

```diff
@@ -161,11 +161,10 @@ templates:
                         attributes:
                             Name: !string {}
                             OfficeAtWeb: *optStr
-            ExclusionOfLiability: !datasource
-                default: []
-                attributes:
-                    Title: !string {} #/definitions/MultilingualText
-                    Content: !string {} #/definitions/MultilingualText
+            DisclaimerGrundbuch_Title: !string {}
+            DisclaimerGrundbuch_Content: !string {}
+            DisclaimerKbS_Title: !string {}
+            DisclaimerKbS_Content: !string {}
```

Comment:
If you change the attribute names like I suggested, then you'll need: 
            DisclaimerLandRegister_Title: !string {}
            DisclaimerLandRegister_Content: !string {}
            DisclaimerPollutedSites_Title: !string {}
            DisclaimerPollutedSites_Content: !string {}

🙂